fedora_messaging/tests/integration/test_publish_consume.py

```diff
@@ -5,11 +5,21 @@
 import threading
 import time
 import unittest
+import uuid
 import socket
 
+from crochet import setup, wait_for
```

Comment:
Why not use pytest_twisted as elsewhere? Did you have issues with it?

Reply:
I wanted to run the actual reactor and start real connections, I think I just assumed twisted_pytest wasn't doing that, but it does seem to just use the normal reactor. I can re-write the tests to use that, I think.

Reply:
Oh, maybe I wanted timeouts so tests wouldn't hang forever? I don't remember.

Reply:
If you think using crochet is better, let's keep it and convert the other tests.

Reply:
In the past I've seen tests pass when they should fail with several of the tools that try to integrate Twisted with unit tests runners. As a result I'm always a bit worried on that subject, we should be very careful that we're using them properly and not causing false negatives.